### DISCUSSION TITLE
CI is using the wrong version yarn

### DISCUSSION THREAD

**Discussion by magic-akari** [swc-project/swc#5614] [General]
Every time I run `yarn install`, I find that `yarn.lock` has changed.
And I found that I was using the wrong version of yarn.
The CI also uses the wrong version of yarn.

I try to add `"packageManager": "yarn@3.2.2"` to `packahe.json`, but another issue ran into.

```
❯ yarn
➤ YN0000: ┌ Resolution step
➤ YN0000: └ Completed
➤ YN0000: ┌ Fetch step
➤ YN0000: └ Completed
➤ YN0000: ┌ Link step
➤ YN0007: │ core-js@npm:2.6.12 must be built because it never has been before or the last one failed
➤ YN0003: │ Some packages have circular dependencies that make their build order unsatisfiable - as a result they won't be built (affected packages are: @swc/core@workspace:.)
➤ YN0000: └ Completed in 3s 659ms
➤ YN0000: Failed with errors in 4s 42ms
```

**Comment by kdy1** [comment]
Oh, now I found the issue, thanks!
I faced it several times, but I didn't dig it.
We may need to remove `@swc/core` from dev deps
